src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -53,3 +53,42 @@
     type: command
     short-summary: Show the price sheet for Azure subscription within a current billing period or specified billing period
 """
+
+helps['consumption marketplace'] = """
+    type: group
+    short-summary: Inspect the marketplace by Azure subscription within current or specified billing period.
+"""
+
+helps['consumption marketplace list'] = """
+    type: command
+    short-summary: Show the marketplace for Azure subscription within a current billing period or specified billing period
+"""
+
+helps['consumption budget'] = """
+    type: group
+    short-summary: Offers ability to create, get, update, and delete budgets for Azure subscription optionally by resource group name
+"""
+
+helps['consumption budget list'] = """
+    type: command
+    short-summary: Lists budget for Azure subscription optionally by resource group name
+"""
+
+helps['consumption budget show'] = """
+    type: command
+    short-summary: Show budget for Azure subscription by budget name and, optionally by, resource group name
+"""
+
+helps['consumption budget create'] = """
+    type: command
+    short-summary: Create budget for Azure subscription and associate with resource group name
```

Comment:
Add `a` before budget. Applies to update/delete as well.

Reply:
Remove `and associate with resource group name`. Applies to update/delete as well.